src/components/MapView/Layers/AdminLevelDataLayer/index.tsx

```diff
@@ -34,7 +35,10 @@ function AdminLevelDataLayers({ layer }: { layer: AdminLevelDataLayerProps }) {
 
   const boundaryId = layer.boundary || firstBoundaryOnView(map);
 
-  const layerData = useSelector(layerDataSelector(layer.id)) as
+  const selectedDate = useDefaultDate(layer.id);
+  // use undefined date for layers without date dimension
+  const date = layer.dateUrl ? selectedDate : undefined;
```

Comment:
I don't think this step is needed. Technically, you could always pass in `selectedDate` and handle the missing layer.dateUrl in `loadLayerData`, no?